### ISSUE TITLE
arXiv.org translator imports original date rather than current revision date

### ISSUE THREAD

**Issue by nschneid** [zotero/translators#1992]
E.g. https://arxiv.org/abs/1810.04805 imports with a date of 2018-10-11 rather than the current version's date of 2019-05-24.

IMO this is harmful because it will be impossible to tell from the Zotero-based bibliography which version is being referenced.

**Comment by zuphilip** [comment]
The translator looks for the XML data e.g. http://export.arxiv.org/oai2?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai%3AarXiv.org%3A1810.04805 and there for DC.date, which in this example looks like
```xml
 <dc:date>2018-10-10</dc:date>
 <dc:date>2019-05-24</dc:date>
```
Current strategy is then to just take the first value. I guess we should rather take the newest date, which we could determine by sorting the strings.

Would it make then sense to save the original date as a separate CSL-field `original-date` which can be saved in the `extra` field?

@nschneid Do you want to fix this issue yourself?

**Comment by adam3smith** [comment]
agree with the first part, but I don't think `original-date` is the right approach for the first date, no. That's typically used for the actual date of publication of a republished paper, not the date of a first version of a manuscript and it'll cause unexpected behavior in (some) citation styles.

**Comment by nschneid** [comment]
> @nschneid Do you want to fix this issue yourself?

I don't have time now, sorry.

**Comment by zuphilip** [comment]
Okay, I will try to look into this and fix it.

> I guess we should rather take the newest date, which we could determine by sorting the strings.

That would work for this example. However, then it would not be possible to cite explicitly the older version. Both versions share the same XML file, which might not be the best to work on... Moreover, the BibTeX in arXiv itself is giving the wrong date in these cases with several versions as well.. Hm... I have to work a little bit more then...

> I don't think original-date is the right approach for the first date, no

Okay, but I try to save additionally the version number, if you are looking at a specific one.

**Comment by zuphilip** [comment]
Fixed in #1995.
